src/pages/components/tile/usage.mdx

```diff
@@ -482,96 +524,39 @@ version.
 
 - Standard layout tiles are the same in height and width as the rest of them in
   the group.
-- A vertical masonry layout can vary in height, but is consistent in width.
+- A vertical masonry layout can vary in height but is consistent in width.
 - A horizontal masonry layout can vary in width. Rows of tiles may vary in
   height, but the tiles within a row should be consistent in height.
 
 <Row>
-<Column colLg={4}>
-
-![Standard layout for tiles](/images/tile-usage-16.png)
-
-<Caption>Standard layout for tiles</Caption>
-
-</Column>
-
-<Column colLg={4}>
-
-![An example of the standard layout](/images/tile-usage-17.png)
-
-<Caption>An example of the standard layout</Caption>
-
-</Column>
-</Row>
-
-<Row>
-<Column colLg={4}>
-
-![Vertical masonry layout for tiles](/images/tile-usage-18.png)
-
-<Caption>Vertical masonry layout for tiles</Caption>
-
-</Column>
-
-<Column colLg={4}>
-
-![An example of the vertical masonry layout](images/tile-usage-19.png)
-
-<Caption>An example of the vertical masonry layout </Caption>
-
-</Column>
-</Row>
-
-<Row>
-<Column colLg={4}>
-
-![Horizonal masonry layout for tiles](/images/tile-usage-20.png)
-
-<Caption>Horizonal masonry layout for tiles</Caption>
-
-</Column>
-
-<Column colLg={4}>
+<Column colLg={8}>
 
-![An example of the horizonal masonry layout](images/tile-usage-21.png)
+![Layouts for tiles](/images/tile-usage-18.png)
 
-<Caption>An example of the horizonal masonry layout </Caption>
+<Caption>
+  Examples of tile layouts: standard, vertical masonry, and horizontal masonry
+</Caption>
 
 </Column>
 </Row>
 
-### Elevation
-
-Tiles reside in the same plane as the background layer—they do not have
-elevation. Do not add drop shadows to tiles. Tiles organize essential
-information and have the same visual hierarchy as content within the same page.
-Modals, popovers, and dialogs do have elevation but are not inherently part of
-the user's main workstream and are invoked on screen to reveal secondary
-information, actions, or notifications.
-
-### Call-to-actions
-
-For **base, selectable, and expandable tiles**, use `$link-01` when icon and
-link are paired together or use `$interactive-04` when the icon stands alone.
-
-For **clickable tiles**, don’t highlight text or icons to avoid them being
-confused as links or clickable.
+## Universal behaviors
 
-<DoDontRow>
-
-  <DoDont type="do" caption="Do use text or icon for clickable tiles if needed.">
-
-![Do use text or icon for clickable tiles if needed.](images/tile-usage-22.png)
+The behaviors listed in this section are universal across all variants. For
```

Comment:
The behaviors listed in this section are universal across all tile variants. For behaviors that are unique to each variant, see each of the component variants sections below.